Gemfile

```diff
@@ -3,6 +3,4 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in librarian.gemspec
 gemspec
 
-gem "fakefs"
```

Comment:
why is this removed?

Reply:
This is already listed in the development dependencies in the gemspec.